frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_list_item.tsx

```diff
@@ -176,6 +180,7 @@ type Props = {
 
 function getSegmentTooltip(segment: Segment) {
   if (segment.creationTime == null) {
+    // TODO: Ask whether the segment name would be better here.
     return `Segment ${segment.id}`;
   }
```

Comment:
I think this function was added before segments had their own names. As I extracted the `getNameOfSegment` method, I would suggest also using it here. But as this hasn't been done previously, there might be a reason for this that I don't know.

So: Should I keep it at the current state or use the new extracted method to show the name of the segment, if it has one?

Reply:
The main purpose of this function is to convey the "creation time" of the segment when hovering it in the segments list (only for very old segments, this information doesn't exist). The id is only mentioned for the sake of completeness (and because it's short). The name doesn't need to be mentioned here in my opinion, because the name is already what the user is currently hovering.